### PR TITLE
Implemented Accuracy Weighted Ensemble

### PR DESCRIPTION
**Pull request by MinhHuong**
Implementation of Accuracy Weighted Ensemble

Changes proposed in this pull request:
- Implementation of Accuracy Weighted Ensemble
- Documentation complies to the format of reStructuredText

@scikit-multiflow/scikit-multiflow-repo-admins

Checklist

- [x] Code complies with PEP-8 and is consistent with the framework.
- [x] Code is properly documented.
- [x] Tests are included for new functionality.
- [x] Travis CI build passes.
- [x] Test Coverage is maintained (threshold 0.2%)

### CODE REVIEW

File: src/skmultiflow/meta/accuracy_weighted_ensemble.py [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+
+            Returns
+            -------
+            boolean
+                true if this object's weight is less than that of the other object
+            """
+            return self.weight < other.weight
+
+    def __init__(self, n_estimators=10, n_kept_estimators=30,
+                 base_estimator=NaiveBayes(), window_size=200, n_splits=5):
```
Comment (AndreFCruz) [inline src/skmultiflow/meta/accuracy_weighted_ensemble.py]: default should be DecisionTreeClassifier according to docstring, right?

Comment (MinhHuong) [reply src/skmultiflow/meta/accuracy_weighted_ensemble.py]: Oh yeah I'm sorry. In fact, DecisionTree isn't available in skmultiflow, and I don't want to mix up both learners from sklearn and skmultiflow, that's why I switched to NaiveBayes (skmultiflow). I will change the docstring accordingly.

Comment (jacobmontiel) [reply src/skmultiflow/meta/accuracy_weighted_ensemble.py]: This comment has been properly addressed.

Thanks @MinhHuong and @AndreFCruz

**Comment by jacobmontiel** [comment]
Thanks @MinhHuong!

I will merge the PR.

**Pull request merged**